.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
I've been removing this line elsewhere so that pull requests against any branch will run CI (for example, against a `purescript-0.15` branch in the future, or...)